src/test/java/dev/hello/utils/TreeUtilTest.java

```diff
@@ -7,6 +7,12 @@
 
 class TreeUtilTest {
 
+    @Test
+    void testTreeUtil() {
+        // keep code coverage from going down
+        new TreeUtil();
```

Comment:
see [static method invocation sites are not covered](https://github.com/jacoco/jacoco/issues/568)